entries/z/zola.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Zola": {
+    "domain": "zola.com",
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "documentation": "https://www.zola.com/expert-advice/a-security-update-for-our-couples",
```

Comment:
```suggestion
```

I wouldn't consider this a documentation page.

Reply:
Yeah, fair. This was the only article I could find on their help site that seemed remotely related to 2FA. Just added a screenshot to the PR description instead.